src/main/java/com/eternalcode/discordapp/command/SayCommand.java

```diff
@@ -0,0 +1,30 @@
+package com.eternalcode.discordapp.command;
+
+import com.jagrosh.jdautilities.command.SlashCommand;
+import com.jagrosh.jdautilities.command.SlashCommandEvent;
+import net.dv8tion.jda.api.Permission;
+import net.dv8tion.jda.api.interactions.commands.OptionType;
+import net.dv8tion.jda.api.interactions.commands.build.OptionData;
+
+import java.util.List;
+
+public class SayCommand extends SlashCommand {
+
+    public SayCommand() {
+        this.name = "say";
+        this.help = "Says something";
+        this.userPermissions = new Permission[]{ Permission.MESSAGE_MANAGE };
+        this.options = List.of(
+                new OptionData(OptionType.STRING, "message", "the message to say")
+                        .setRequired(true)
+        );
+    }
+
+    @Override
+    public void execute(SlashCommandEvent event) {
+        String message = event.getOption("message").getAsString();
+
+        event.getChannel().sendMessage(message).queue();
```

Comment:
Jak tak myślę, to można byłoby wprowadzić jeszcze opcję wysłania tego jako embed i fajne by było to że jak wykonujesz tą komendę jako odpowiedź do kogoś, to bot odpowiada tej osobie, jeśli wiesz co mam na myśli